### DISCUSSION TITLE
Inquiry into the Adaptability of the L2 Building Solution for L3 Development

### DISCUSSION THREAD

**Discussion by abitocodes** [gokch/optimism_builder#10] [Q&A] [answered]
Hello! I deeply appreciate your remarkable contributions to open-source projects.

I'm currently exploring the L2 building solution, which is primarily designed for L1-L2 integration. However, I'm curious to know if you think it would technically be feasible to extend this solution for L2-L3 development to create L3 applications. 

Although you might not have tried this specifically, I would value your insight on whether you see any potential technical issues with such an application. 

Thank you!

**Comment by gosunuts** [comment, marked as answer]
Thanks for your interest in builderism! L3 support is an important feature in builderism as well.
Unfortunately, there is very little information about deploying L3 chains.

I found some articles.
- https://github.com/ethereum-optimism/developers/discussions/407
- https://github.com/ethereum-optimism/specs/discussions/158
- https://stack.mirror.xyz/l9w2lQ9MEeCkLm0PDGubogSEuQhL1ETjqg6BIs4U76k
- https://app.conduit.xyz/deploy?framework=OPTIMISM&environment=TESTNET&settlement=BASE&dataAvailability=CELESTIA

It seems possible, but more research and testing is needed. If you're interested, feel free to contribute!
